jenkins-agents/jenkins-agent-mvn/Dockerfile

```diff
@@ -1,2 +1,29 @@
+#FROM quay.io/openshift/origin-jenkins-agent-maven:4.6
+#ADD settings.xml $HOME/.m2/settings.xml
+#WORKDIR $HOME
+#COPY pom.xml ./
+#RUN mvn -e org.apache.maven.plugins:maven-dependency-plugin:2.10:tree -Dverbose=true
+#RUN mvn -e dependency:go-offline -B
+#RUN mvn package 
+#USER 1001
+
 FROM quay.io/openshift/origin-jenkins-agent-maven:4.6
-ADD settings.xml $HOME/.m2/settings.xml
+ADD settings.xml $HOME/settings.xml
```

Comment:
Why did we change this from `$HOME/.m2/settings.xml` to `$HOME/settings.xml` ?

Reply:
![image](https://user-images.githubusercontent.com/53479552/148899962-a77d8df9-68e4-4d59-92ac-cc9dc52feb48.png)
  
With the .m2 folder getting this error , so removed it.

Reply:
I don't think we can remove it. Can you check how to fix the error without removing it ?

Reply:
I have reset the HOME variable path as the path set in the image is corrupted and does not provide access to download all plugins required. It threw error even after adding the plugin in pom.xml which is as per maven document. 

After resetting the HOME variable for the image, the error resolved. We could use the .m2 folder for storing all dependencies .

Reply:
![image](https://user-images.githubusercontent.com/53479552/149140954-04f00caa-84ce-4c30-b257-870e3c8de0c7.png)